### PR TITLE
Release notes v2.5.0: STM32 and shields additions

### PR DESCRIPTION
**Pull request by erwango**

### CODE REVIEW

**Review by nashif — APPROVED** [review]

File: doc/releases/release-notes-2.5.rst [hunk 1]
```diff
@@ -194,12 +223,15 @@ Boards & SoC Support
...
 
   * Inventek es-WIFI shield
+  * Sharp memory display generic shield
 
 Drivers and Sensors
 *******************
 
 * ADC
 
+  * A sample is now available to demonstrates how to use the ADC driver API.
```
Comment (ioannisg) [inline doc/releases/release-notes-2.5.rst:233]: Use past tense here, as well.

Comment (erwango) [reply doc/releases/release-notes-2.5.rst]: Fixed, let me know if this is fine

**Review by gmarull — COMMENTED** [review]
Support for FMC/SDRAM should also be mentioned

**Comment by erwango** [comment]
> Support for FMC/SDRAM should also be mentioned

Thanks! Suprised I missed it. I hope I didn't miss much more.

**Comment by gmarull** [comment]
@erwango actually, support for stm32cubeprogrammer should also be on the list

**Comment by erwango** [comment]
@gmarull Fixed, and txs again for the great work!

**Review by gmarull — APPROVED** [review]

**Pull request merged**